testing/kfctl/conftest.py

```diff
@@ -21,6 +21,10 @@ def pytest_addoption(parser):
       "--use_basic_auth", action="store", default="False",
       help="Use basic auth.")
 
+  parser.addoption(
+      "--use_istio", action="store", default="False",
```

Comment:
shouldn't you use something like this?
https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/get_kf_testing_cluster.py#L229-L237

Reply:
it's a different module called pytest. See above for basic auth